src/OrchardCore.Modules/OrchardCore.Title/Handlers/TitlePartHandler.cs

```diff
@@ -40,6 +40,16 @@ public override Task CreatedAsync(CreateContentContext context, TitlePart part)
         return SetTitleAsync(part);
     }
 
+    public override Task ClonedAsync(CloneContentContext context, TitlePart part)
+    {
+        return SetTitleAsync(part);
+    }
+
+    public override Task ImportedAsync(ImportContentContext context, TitlePart part)
+    {
+        return SetTitleAsync(part);
+    }
```

Comment:
```suggestion
        => SetValueAsync(context, field);
```

Reply:
> I’d like to suggest we stop enforcing style preferences unless we have an explicit rule in place

>  let's respect that and allow contributors to use the style they prefer

I disagree, so that everyone will use their own style, which means there's no sense for the current style that we spent time on. 

@sebastienros @Piedone @Skrypt, your thought?

Reply:
We absolutely shouldn't use personal preferences, style or otherwise, _as long as there's a common standard_ (which for this can be in `.editorconfig`, there's some code style rule for expression-bodied members). https://github.com/OrchardCMS/OrchardCore/issues/7950 includes things like this, so if you're interested, please work on it, I can provide help.

Reply:
If there is an analyzer to enforce a rule, I am all in otherwise we should not convert a PR review into a formatting review and forget about the actual change that is being introduced. 

Leave format for analyzer and code style while PR reviewers should only worry about the code changes.

Reply:
> Leave format for analyzer and code style while PR reviewers should only worry about the code changes.

Right, but if the author does not follow the guidelines, we should request a change. I saw many PRs from different contributors over that time that have formatting issues in medium to long PRs, which makes the review take so long, but again, we need to follow the guidelines